cmake/external/codecov.cmake

```diff
@@ -19,4 +19,4 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang|GNU")
     endif()
 else()
     message(STATUS "Compiler ${CMAKE_CXX_COMPILER_ID} doesn't support code-coverage.")
-endif()
+endif()
```

Comment:
Unneeded change. Please add new line at the end of file and configure your IDE to always add a new line at the end of file. In VSCode it can be done by settings `"editor.formatOnSave": true` (or similar property).